lib/cloudinary.js

```diff
@@ -0,0 +1,9 @@
+require("dotenv").config();
```

Comment:
This should be in the server.js, then you only have to import it once

Reply:
👍

Reply:
Thanks for reviewing!